site/docs/try/releases.md

```diff
@@ -1,5 +1,9 @@
 # Releases
 
+## 0.10.0 Release (August xx, 2021)
+
+* New tiered-version-store implementation
```

Comment:
nit: is it still tiered?

Reply:
We'll re-work the release-notes for the next release anyway.